docs/Doxyfile

```diff
@@ -806,8 +806,12 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = ../libminifi/src \
-                         ../libminifi/include
+INPUT                  = ../main \
```

Comment:
we renamed `main` to `minifi_main` in #1394

Reply:
Updated in 18d2c7a6b3b3fdaf802439933beba79eab8334d8